### PR TITLE
Updated contribution guide to be similar to React's

### PR DESCRIPTION
**Pull request by tomarak**
I updated the contribution guide to be similar to the one found in the React repo:
https://github.com/facebook/react/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,27 +10,50 @@ We actively welcome your pull requests.
...
+
+### Reporting New Issues
+
+The best way to get your bug fixed is to provide a reduced test case. jsFiddle, jsBin, and other sites provide a way to give live examples. Those are especially helpful though may not work for `JSX`-based code.
+
+### Security Bugs
+
+Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe disclosure of security bugs. With that in mind, please do not file public issues; go through the process outlined on that page.
+
+## Style Guide
```
Comment (kyldvs) [inline CONTRIBUTING.md]: I think this section needs to be removed, there is no `grunt lint` step in this project.

**Comment by kyldvs** [comment]
If you can change that this looks ready to merge!

**Comment by tomarak** [comment]
@kyldvs done! nearly a year later...:)

**Comment by kyldvs** [comment]
And merged just about another year later... :) Nice teamwork! :P

**Pull request merged**